projects/igniteui-angular/src/lib/slider/slider.component.ts

```diff
@@ -1435,7 +1435,7 @@ export class IgxSliderComponent implements
     }
 
     private emitValueChanged(oldValue: number | IRangeSliderValue) {
```

Comment:
can you consider renaming this to emitValueChange, so it is compliant with the event?